### ISSUE TITLE
How to fix this error: U-blox: received NACK: 0x06 /0x01

### ISSUE THREAD

**Issue by QuinnZhuu** [KumarRobotics/ublox#56]
I have one GPS receiver with U-BLOX M8030 72CH , but when I run the launch file. This error occurred.
![IMG_3143](https://user-images.githubusercontent.com/41944128/61463527-2496b580-a9a7-11e9-98bb-1ccb837fd11e.JPG)

I wonder how to solve this problem
Thanks!

**Comment by CK-Sung** [comment]
Have you solved this problem? I have same error "received NACK: 0x06 0x01".

**Comment by SilvainLouis** [comment]
Same probleme when I try to enable other than GPS system

**Comment by JihunDev** [comment]
I am developing using ROS2.

The error "U-blox: received NACK: 0x06 /0x01"  occurs.

but check it using ROS2 Topic, data comes in.

If you have solved this problem, please share it.

**Comment by NovoG93** [comment]
Anyone has an update on this?

**Comment by vilktor370** [comment]
Yeah i can get data but sometimes to data is 0.0 in both latitude and longitude. So I am not sure what is going on here :(

![Screenshot from 2021-07-27 12-39-55](https://user-images.githubusercontent.com/66043167/127193760-dbe90e51-7cc7-479b-8104-71f733211f13.png)

**Comment by trayveet** [comment]
I had similar issues. I was able to trace it the USB HUB I was using. Using a different hub fixed that problem, but other issues arised as well. 
Hope it helps

**Comment by tomy983** [comment]
```
publish:
    aid/all: false
```
instead of the incorrect 
```
publish:
    aid: false
```
fixed it for me with ZED F9P. Find if you are trying to publish incorrect messages...

**Comment by gokulp01** [comment]
Has anyone fixed this issue? This is persistent in both ros2 foxy and ros2 galactic. Any idea how to fix it?

Help will be much appreciated, thanks!

**Comment by gokulp01** [comment]
Update: I fixed the issue. The issue is in the config file. Make sure to use the right config file. (I also updated the firmware through the U-blox center, but I don't think that was very helpful)
